apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -181,16 +182,17 @@
                   description="When set only show when javascript expression is !null or !false. Example: other > 5"
                 />
               </div>
-              <div class="col-4">
+              <div class="col-4" v-if="!requiredSelect">
```

Comment:
might this not be confusing that computed cannot be seen/edited? How do we explain this to the user? We could disable or remove the input and instead show a message that 'computed' cannot be edited unless required is false?